js/main.js

```diff
@@ -19,15 +13,15 @@ function switchView(dataView) {
 
 function switchViewing(event) {
   event.preventDefault();
-  var dataView = event.target.getAttribute('data-view');
-  switchView(dataView);
+  var $viewSwapper = document.querySelectorAll('.view-swap');
+  for (var i = 0; i < $viewSwapper.length; i++) {
+    if ($viewSwapper[i] === event.target) {
+      var dataView = $viewSwapper[i].getAttribute('data-view');
+      switchView(dataView);
+    }
+  }
```

Comment:
Done this way, you don't need a loop, you just need to check if the `event.target` element has the class `view-swap`. If it does, you want to call the `switchView` function with the value of that `data-view` attribute